distribution/changelog.txt

```diff
@@ -70,6 +70,7 @@
 - Fix: [#17904] Trees are removed at no cost to make way for ride entrance/exits.
 - Fix: [#17905] The chain button in the map window is enabled for rectangular maps when (re)opened.
 - Fix: [#17931] The in-game command ‘count_objects’ crashes the game.
+- Fix: [#17853] Invention name tears while being dragged.
```

Comment:
0.4.2 is already released. You need to rebase this on the current `develop` branch and move it to the 0.4.3 part.

Reply:
Oh yeah didn't know that. Just moved it to under 0.4.3. Thanks!